### PR TITLE
Theses can be edited by admin, but belong to user

### PR DESCRIPTION
**Pull request by criedlberger**

### CODE REVIEW

File: spec/controllers/my/files_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        expect(assigns[:document_list].map(&:id)).to include(@my_file.id)
+        # doesn't show files shared with other users
+        expect(assigns[:document_list].map(&:id)).to_not include(@unshared_file.id)
+      end
+  end
+
+  describe 'User can edit and not edit' do
+    before do
+      visit "/dashboard/files"
+
```
Comment (pgwillia) [inline spec/controllers/my/files_controller_spec.rb]: This before doesn't have a closing end so the following tests aren't executed (should be 4 examples, only registers two).

File: spec/controllers/my/files_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        expect(page).to have_content( "Edit File" )
+      end
+
+      it 'user cannot edit' do
+        click_link "My Files"
+        within("#document_#{@read_shared_with_me.id}") do
+          click_button("Select an action")
+        end
+        expect(page).to_not have_content( "Edit File" )
+      end
```
Comment (pgwillia) [inline spec/controllers/my/files_controller_spec.rb]: These are good feature styled tests (click_link, click_button, etc) which test the view rather than just the controller so should be in spec/features not spec/controllers.

File: spec/controllers/my/all_controller_spec.rb [hunk 3]
```diff
@@ -43,34 +43,6 @@
...
-        expect(assigns[:document_list].map(&:id)).to_not include(@my_collection.id)
-        # doesn't show shared files
-        expect(assigns[:document_list].map(&:id)).to include(@shared_file.id)
-        # doesn't show other users' files
-        expect(assigns[:document_list].map(&:id)).to_not include(@unrelated_file.id)
-        # doesn't show non-generic files
-        expect(assigns[:document_list].map(&:id)).to_not include(@wrong_type.id)
-      end
-    end
-  end
```
Comment (pgwillia) [inline spec/controllers/my/all_controller_spec.rb]: First thought is why did you remove the old test? 

Second thought is don't we want to verify that non-admin users get an unsuccessful response?

File: spec/controllers/my/files_controller_spec.rb (hunk 2 above)
Comment (pgwillia) [inline spec/controllers/my/files_controller_spec.rb]: Is there a test for the I Can Edit / Depositor: me case?

File: spec/controllers/my/files_controller_spec.rb (hunk 1 above)
Comment (criedlberger) [inline spec/controllers/my/files_controller_spec.rb]: Corrected

File: spec/controllers/my/files_controller_spec.rb (hunk 2 above)
Comment (criedlberger) [inline spec/controllers/my/files_controller_spec.rb]: I have added the test.

File: spec/controllers/my/all_controller_spec.rb (hunk 3 above)
Comment (criedlberger) [inline spec/controllers/my/all_controller_spec.rb]: Non-admin users should not be seeing the All Files tab at all.

**Comment by pgwillia** [comment]
Looks good.  Waiting on travis.
